src/main/java/core/basesyntax/strategy/impl/SupplyOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;

```

Comment:
Classes within the 'strategy' package should not directly access the 'Storage' class. Instead, consider creating a service interface and its implementation that interacts with the 'Storage' and use it here. This will adhere to the SOLID principles better, specifically the Dependency Inversion Principle.